deploy/lib/xquery/setup.xqy

```diff
@@ -4544,6 +4559,13 @@ declare function setup:get-privilege-by-name($name as xs:string) as element(sec:
     </options>)
 };
 
+declare function setup:get-users-by-name($names as xs:string*) as element(sec:users)? {
+  let $ids :=
+    for $name in $names
+      return xdmp:user($name)
```

Comment:
I agree, I will make change to absorb when users and roles don't exists. Thanks!